clang/include/clang/Driver/Options.td

```diff
@@ -3621,7 +3621,14 @@ defm sycl_esimd_force_stateless_mem : BoolFOption<"sycl-esimd-force-stateless-me
             "Disabled by default. (experimental)">,
     NegFlag<SetFalse, [], [ClangOption], "Do not enforce using stateless memory accesses. (experimental)">,
     BothFlags<[], [ClangOption, CLOption, DXCOption, CC1Option], "">>;
-
+// TODO: Remove this option once ESIMD headers are updated to
+// guard vectors to be device only.
+defm sycl_esimd_build_host_code : BoolFOption<"sycl-esimd-build-host-code",
+    LangOpts<"SYCLESIMDBuildHostCode">, DefaultTrue,
+    PosFlag<SetTrue, [], [ClangOption], "Build the host implementation of ESIMD functions."
+            "Enabled by default.">,
```

Comment:
@sarnex, thinking about this a bit more, I don't think that we should be introducing a positive flag at all. If the option is a temporary thing, which is supposed to go away together with ESIMD emulator (or at some other point), I don't see a value in introducing the positive flag here. It is just a one more thing which users will have to adjust to if they start using it.

Reply:
Yeah this is a good point, I didn't want to add a positive option either originally but I got the idea there was an informal rule to have one. I don't think that's true and it doesn't make sense here anyway. I'll make a PR to remove it.